index.html

```diff
@@ -5,6 +5,22 @@
   <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
   <meta content="utf-8" http-equiv="encoding">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="Try PureScript is an online PureScript code editor for quickly experimenting with PureScript code snippets and ideas.">
+
+  <!--  Facebook  -->
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="http://try.purescript.org/">
+  <meta property="og:title" content="Try PureScript!">
+  <meta property="og:image" content="https://raw.githubusercontent.com/purescript/logo/master/youtube/channel_icon-800.png">
```

Comment:
Maybe we should copy this image into this repo?

Reply:
Done. 🙌